src/app/pages/admin/search/search-form.component.ts

```diff
@@ -73,6 +79,11 @@ export class SearchFormComponent {
       (_, index) => this.searchForm.value.modules?.[index]
     );
   }
+
+  public isSchool(): boolean {
```

Comment:
Given that we're using this method to evaluate an expression on a constant and a field that has been set in the constructor, exposing a field instead of a method could be an alternative:
```
public readonly currentRoleIsSchool;
```
```
this.currentRoleIsSchool= this.permissionController.getRoleType() === Roles.School;
```
In that way we don't need the method.

Reply:
implemented